spec/features/create_new_album_spec.rb

```diff
@@ -57,10 +57,10 @@
       expect(page).to have_content "Vacation Pics"
       expect(page).to have_content "These are pictures of my family on vacation!"
       within('#photo0') do
-        expect(find('img')['src']).to have_content "Airturbine.jpg"
+        expect(find('img')['src']).to have_content "ice-boats.jpg"
       end
       within('#photo1') do
-        expect(find('img')['src']).to have_content "ice-boats.jpg"
+        expect(find('img')['src']).to have_content "Airturbine.jpg"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.